ansible/roles/ecryptfs/tasks/main.yml

```diff
@@ -34,7 +34,7 @@
 
 - name: Register mount configuration
   sudo: yes
-  shell: "tail /etc/mtab -n1"
+  shell: "cat /etc/mtab | grep '/opt/data' | sed -e 's/ rw/ user,noauto,rw/'"
```

Comment:
The issue is that the line we're adding is coming from the output of `tail /etc/mtab` but we want to alter the line before writing it to the file.